lib/components/viewers/pattern-row.tsx

```diff
@@ -88,7 +88,7 @@ class PatternRow extends Component<Props, State> {
           <div className="route-name">
             <strong
               style={
-                routeName && routeName?.length >= 4 ? { fontSize: '3vb' } : {}
+                routeName && routeName?.length >= 4 ? { fontSize: '50%' } : {}
```

Comment:
This or similar logic also needs to be applied to the related stops rows (see screenshot)

<img width="375" alt="Screenshot 2022-11-21 at 3 29 39 PM" src="https://user-images.githubusercontent.com/56846598/203152469-2bb5b367-bae0-4275-bbe5-477a81d78867.png">

Reply:
Spent a long time trying to get the grid alignment right while also doing dynamic text sizing, in the end decided to just un-grid it. 31f58ccb70b6b8e55470e28c9eb85f09a8139dc3